rcsb/workflow/bcif/task_functions.py

```diff
@@ -0,0 +1,305 @@
+##
+# File:    task_functions.py
+# Author:  James Smith
+# Date:    21-Feb-2025
+##
+
+"""
+Workflow task descriptors.
+"""
+
+__docformat__ = "google en"
+__author__ = "James Smith"
+__email__ = "james.smith@rcsb.org"
+__license__ = "Apache 2.0"
+
+import multiprocessing
+import os
+import shutil
+import tempfile
+import logging
+from enum import Enum
+from typing import Optional
+from mmcif.api.DictionaryApi import DictionaryApi
+from mmcif.io.IoAdapterPy import IoAdapterPy as IoAdapter
+from rcsb.utils.io.MarshalUtil import MarshalUtil
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+ContentTypeEnum = Enum(
```

Comment:
@piehld would you know if we have this enum elsewhere so that we can reuse it?

Reply:
Oh yes... definitely that one is not reusable. Then let's keep it as it is. Thanks